src/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Header from '../../components/Header/Header';
+import './HomePage.css';
+import SearchBar from '../../components/SearchBar/SearchBar';
+import CardsList from '../../components/CardsList/CardsList';
```

Comment:
Import path casing should match exactly.
'../../components/cardsList/CardsList' folder 'cardsList' should be in lower case.

Reply:
The same